.github/workflows/devcontainer.yaml

```diff
@@ -0,0 +1,63 @@
+name: devcontainer
+
+on:
+  push:
+    # ToDo: Add partial Taskfile (only include rust-tools)
```

Comment:
We should indeed add `Taskfile.yml`. 

It's fine to run workflows too often, and we're never going to get it precise, so we should lean on over-building.